aiidalab_widgets_base/computational_resources.py

```diff
@@ -728,13 +732,8 @@ def _configure_computer(self):
             authparams["proxy_command"] = sshcfg["proxycommand"]
         elif "proxyjump" in sshcfg:
             authparams["proxy_jump"] = sshcfg["proxyjump"]
-        aiidauser = orm.User.objects.get_default()
 
-        authinfo = orm.AuthInfo(
-            computer=orm.Computer.objects.get(label=self.label.value), user=aiidauser
-        )
-        authinfo.set_auth_params(authparams)
-        authinfo.store()
+        computer.configure(**authparams)
```

Comment:
So this function does what the previous code did w.r.t to `authinfo`?

Reply:
Well, here I incorporated a function provided by AiiDA that does exactly the same. Just a little simplification.

I will now modify the PR description to better describe what has happened here.

Reply:
done.

Reply:
Thanks!